cpp/example_code/s3/tests/gtest_create_bucket.cpp

```diff
@@ -19,8 +19,7 @@ namespace AwsDocTest {
 // NOLINTNEXTLINE(readability-named-parameter)
     TEST_F(S3_GTests, create_bucket_2_) {
         Aws::String uuid = Aws::Utils::UUID::RandomUUID();
-        Aws::String bucketName = "amzn-s3-demo-bucket-" +
-                                 Aws::Utils::StringUtils::ToLower(uuid.c_str());
+        Aws::String bucketName = GetUniqueBucketName();
```

Comment:
I assume this C++ function returns a unique bucket name that meets standards.

Reply:
Yes, it takes a prefix from an environment variable. And this is a test.